### ISSUE TITLE
<br> tags are added after update to 6.2.2

### ISSUE THREAD

**Issue by errotu** [publishpress/PublishPress-Authors#1348]
## Expected Behavior

The "View all posts" link should be displayed in the author box. 

```
<a href="https://domain.com/author/joe-doe/" title="View all posts"><br>
	<span>View all posts</span><br>
</a>
```

## Current Behavior

The "View all posts" is not visible anymore, as it is written in white and moved outside the "button" with colored background by <br> tags. Now, the white font is displayed on the white background below the button.

```
<a href="https://domain.com/author/joe-doe/" title="View all posts">
	<span>View all posts</span>
</a>
```

## Possible Solution

Find out what causes the br tags and remove them.

A possible (dirty) quick fix is to disable br tags via CSS:

```
.pp-author-boxes-meta br {
  display: none;
}
```

## Steps to Reproduce (for bugs)

1. Use the author box via a shortcode in a page template (Gutenberg Full-Site-Editing)
2. Update to Wordpress v6.2.2
3. br tags occur which didn't before

## Context

As written above, the error occurred after an update to v6.2.2. I am not sure if it only affects author boxes added by shortcodes in a page template (Gutenberg Full-Site-Editing) or all author boxes independent of the way they are integrated. Please note that v6.2.1 and v6.2.2 [concern security issues with shortcodes in page templates](https://core.trac.wordpress.org/ticket/58333).

**Comment by stevejburge** [comment]
@errotu we don't see this issue now. Do you think it was related to the shortcodes issues with 6.2.1?

**Comment by errotu** [comment]
Yes, I think it's related to the shortcode issue. It only happens with author boxes which are added via shortcode in page _templates_ (using Gutenberg's Full Site Editing). When I add the author box via shortcode as part of a page (and not a template), there are no additional br-tags.
